src/emulator/modules/SceTouch/SceTouch.cpp

```diff
@@ -64,6 +149,10 @@ static int peek_touch(HostState &host, SceUInt32 port, SceTouchData *pData, SceU
         }
     }
 
+    else if (registered_touch() == true && port == touchscreen_port) {
```

Comment:
Weird newline

Reply:
peek_touch is always polling so only recover the touches if any touches are registered. also its polling on both the back and front touch panel so i filter it out with port == touchscreen_port so it doesn't touch both simultaneously

Reply:
By new line i mean why this exist

![image](https://user-images.githubusercontent.com/25717050/58741617-54184f80-8445-11e9-9ba1-9827f261ae9f.png)

Reply:
i think i have to run format.sh again

Reply:
its actually still there, no idea why